apps/condo/domains/organization/schema/OrganizationEmployee.js

```diff
@@ -122,6 +122,16 @@ const OrganizationEmployee = new GQLListSchema('OrganizationEmployee', {
             defaultValue: false,
         },
     },
+    kmigratorOptions: {
+        constraints: [
+            {
+                type: 'models.UniqueConstraint',
+                fields: ['user', 'organization'],
+                condition: 'Q(deletedAt__isnull=True)',
+                name: 'unique_user_and_organization',
```

Comment:
Please add schema name in name, since it can possible be multiple org + user combos in different schemas